.circleci/config.base.yml

```diff
@@ -194,7 +194,7 @@ jobs:
           environment:
             JEST_JUNIT_OUTPUT: 'reports/junit/js-test-results.xml'
       - store_test_results:
-          path: packages/amplify-util-mock/
+          path: ~/repo/packages/amplify-util-mock/
```

Comment:
what does this (and similar) change do ?

Reply:
~ is the most reliable way to resolve paths in the CircleCI executor, and is consistent between linux & windows (they use some kind of interpretation to resolve ~ in both linux and windows)

The scan_artifacts.ts file exists in ~/repo/.circleci/scan_artifacts.ts, and we can easily determine the absolute value of ~ by generating a path that moves up 2 directories:
const REPO_FOLDER = path.normalize(path.join(__dirname, '..', '..')); // same as ~

I am also using normalize to make sure that slashes are correct, based on the OS:
const normalizedFolder = path.normalize(folder);

Using ~ is mainly for simplicity; as I don't want people to be concerned with absolute paths, since all they need to know is that the working directory is ~, and everything exists inside of repo 
repo/src/packages/amplify-e2e-tests/.. etc